wbooks/src/App/Main/BookDetail/CommentsSection/Comment/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './styles.css';
+
+class Comment extends React.Component {
```

Comment:
Instead of mocking this component, change it so it receives the props (message, user, etc).

Also use prop-types for validation.
https://reactjs.org/docs/typechecking-with-proptypes.html